admin-dev/themes/new-theme/template/components/layout/mobile_quickaccess.tpl

```diff
@@ -0,0 +1,47 @@
+<div class="component-search-quickaccess d-none">
+  <p class="component-search-title">{l s='Quick access' d='Admin.Navigation.Header'}</p>
```

Comment:
This string looks new (without the capital A). Do you know where it appears exactly? I don't see it on your screenshots.

Reply:
It's the quick access title on the screen but uppercase comes from the style